include/Entities/Enemy1.hpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Ruoying Hao on 10/12/17.
+//
+
+#ifndef ENEMY1_HPP
+#define ENEMY1_HPP
+#include "Entities/Entity.hpp"
+
+class Enemy1 : public Entity{
+public:
+    Enemy1();
+    bool isHacked();
+    void setStartPos(sf::Vector2f pos);
+    void setDestPos(sf::Vector2f pos);
+    sf::Vector2f getStartPos();
+    sf::Vector2f getDestPos();
+    //part of path finding, find the next position that the entity should go to
+    sf::Vector2f getNextPos();
+    static const float COLLISION_SIZE;
+
+private:
+    bool hacked;
```

Comment:
If both entity1 and entity2 have these private variables and public methods, why not put them in a super class?